package.json

```
@@ -15,7 +15,9 @@
     "build-storybook": "build-storybook",
     "db:create-users-table": "ts-node ./src/scripts/create_users_table.ts",
     "db:delete-users-table": "ts-node ./src/scripts/delete_users_table.ts",
-    "db:get-user-from-sub": "ts-node ./src/scripts/get_user_from_sub.ts"
+    "db:get-user-from-sub": "ts-node ./src/scripts/get_user_from_sub.ts",
+    "e2e-addRedirectUri": "ts-node bin/addRedirectURItoConnect.ts",
+    "e2e-removeRedirectUri": "ts-node bin/removeRedirectURIFromConnect.ts"
```

Do you think we should keep consistency in our script naming? Something like:
```suggestion
    "e2e:add-redirect-uri": "ts-node bin/add-redirect-uri-from-application.ts",
    "e2e:remove-redirect-uri": "ts-node bin/remove-redirect-uri-from-application.ts"
```
That way, if we see kebab casing, we know it's a "self invoked" function.

---

I agree, let's do that ! It will be more clear that way.